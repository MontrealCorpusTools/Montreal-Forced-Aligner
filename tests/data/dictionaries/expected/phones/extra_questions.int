1 2 3 4 5 6 7 8 9 10
11 12 13 14 15 16 17 18 19 20 21 22
11 15 19
12 16 20
13 17 21
14 18 22
1 6
2 7
3 8
4 9
5 10
