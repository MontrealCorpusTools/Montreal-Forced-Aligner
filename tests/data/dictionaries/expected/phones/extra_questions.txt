sil sil_B sil_E sil_I sil_S spn spn_B spn_E spn_I spn_S
phonea_B phonea_E phonea_I phonea_S phoneb_B phoneb_E phoneb_I phoneb_S phonec_B phonec_E phonec_I phonec_S
phonea_B phoneb_B phonec_B
phonea_E phoneb_E phonec_E
phonea_I phoneb_I phonec_I
phonea_S phoneb_S phonec_S
sil spn
sil_B spn_B
sil_E spn_E
sil_I spn_I
sil_S spn_S
